src/Portal.cpp

```diff
@@ -18,6 +18,52 @@ char Portal::getTipoDePortal() {
 	return this->tipoDePortal;
 }
 
-CoordenadaParcela * Portal::getCoordenadaAsociada() {
-	return this->parcelaAsociada;
+void Portal::accionarPortal(bool nace, RGB* color){
+	if(this->tipoDePortal == ACTIVO) this->accionarPortalActivo(nace, color);
+	else if(this->tipoDePortal == NORMAL) this->accionarPortalNormal(nace, color);
+	else if(this->tipoDePortal == PASIVO) this->accionarPortalPasivo(nace, color);
+
+}
+
+void Portal::accionarPortalActivo(bool nace, RGB* color){
+	if(this->portalOrigen){
+		if(nace) this->nacioOrigenEntoncesNaceDestino(color);
+		else this->murioOrigenEntoncesMuereDestino();
+	}
+	else if (!nace) this->murioDestinoEntoncesMuereOrigen();
+}
+
+void Portal::accionarPortalNormal(bool nace, RGB* color){
+	if(this->portalOrigen){
+		if(nace) this->nacioOrigenEntoncesNaceDestino(color);
+		else this->murioOrigenEntoncesMuereDestino();
+	}
+}
+
+void Portal::accionarPortalPasivo(bool nace, RGB* color){
+	if(this->portalOrigen){
+		if(nace) this->nacioOrigenEntoncesNaceDestino(color);
+	}
+}
+
+void Portal::nacioOrigenEntoncesNaceDestino(RGB* color){
+	int fila = this->parcelaAsociada->getCoordenadaX();
+	int columna = this->parcelaAsociada->getCoordenadaY();
+	float factorNacimiento = this->parcelaAsociada->getTablero()->getParcela(fila, columna).getfactorNacimiento();
+	this->parcelaAsociada->getTablero()->getParcela(fila,columna).getCelula()->nacer(factorNacimiento,color);
```

Comment:
`Tablero * tablero = this->parcelaAsociada->getTablero()`
y dsp
`tablero->getParcela(fila, columna).....`
`tablero->getDatosTablero()....`